capstone/fitness/src/main/java/com/google/sps/fit/Exercise.java

```diff
@@ -0,0 +1,211 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.fit;
+
+import java.io.Serializable;
+import java.util.*;
+
+/* Representation of exercise with specific quantitative data. */
+public class Exercise implements Serializable {
+  
+  public enum SetType {
+    DISTANCE,
+    DURATION_INC,
+    DURATION_DEC,
+    REPS,
+    WEIGHT;
+    
+    /**
+     * Certain measurements for exercises want to decrease in number whereas others don't. This addresses that by
+     * returning a "greater than" comparison for two numbers depending on if more of the stat is better.
+     *
+     * @param src starting float
+     * @param comp float to compare to
+     * @return if the starting float is "greater than" its comparison.
+     */
+    public boolean greaterThan(float src, float comp) {
+      if(this.name().contains("_DEC")) {
+        return src < comp;
```

Comment:
done; I changed it to betterThan instead of isGoalReached because I wanted to separate the Exercise logic from the GoalStep logic (isGoalReached sounds like it would be in GoalStep) while still having a comparison between objects. I feel "betterThan" represents the language I was going for initially.